safekeeper/src/bin/safekeeper.rs

```diff
@@ -143,19 +137,27 @@ fn main() -> anyhow::Result<()> {
 }
 
 fn start_safekeeper(mut conf: SafeKeeperConf, given_id: Option<NodeId>, init: bool) -> Result<()> {
-    let log_file = logging::init("safekeeper.log", conf.daemonize, conf.log_format)?;
-
+    logging::init(conf.log_format)?;
     info!("version: {GIT_VERSION}");
 
     // Prevent running multiple safekeepers on the same directory
     let lock_file_path = conf.workdir.join(LOCK_FILE_NAME);
-    let lock_file = File::create(&lock_file_path).context("failed to open lockfile")?;
```

Comment:
cc @petuhovskiy , have changed SK lockfile a bit: now it's a `safekeeper.pid` file in the same directory and it contains a pid of the process.

LMK if there are some concerns related.